parlai/core/teachers.py

```diff
@@ -1084,3 +1087,90 @@ def setup_data(self, path):
                     reward = 0
             if x:
                 yield [x, None, reward], start
+
+
+class ParlAIDialogTeacher(FixedDialogTeacher):
+    """This module provides access to data in the ParlAI Text Dialog format.
+
+
+    Subclasses ``DialogTeacher`` for functionality and provides an
```

Comment:
It actually subclasses `FixedDialogTeacher`

Reply:
fixed